qiskit/circuit/random/utils.py

```diff
@@ -21,7 +21,14 @@
 
 
 def random_circuit(
-    num_qubits, depth, max_operands=4, measure=False, conditional=False, reset=False, seed=None
+    num_qubits,
+    depth,
+    max_operands=None,
+    num_operand_distribution: dict = None,
```

Comment:
Please do not change the order of positional arguments per @1ucian0 comment!

Reply:
I have made the `num_operand_distribution` argument as the last.